api/src/main/java/org/pentaho/platform/api/genericfile/model/IGenericFileTree.java

```diff
@@ -43,4 +43,21 @@ public interface IGenericFileTree {
    */
   @Nullable
   List<IGenericFileTree> getChildren();
+
```

Comment:
Consider saying that the list returned by `getChildren()` (when not null) can be modified directly.